apps/api/src/courses/schemas/showCourseCommon.schema.ts

```diff
@@ -20,6 +20,8 @@ export const commonShowCourseSchema = Type.Object({
   isScorm: Type.Optional(Type.Boolean()),
   priceInCents: Type.Number(),
   thumbnailUrl: Type.Optional(Type.String()),
```

Comment:
For now I also left the old value because it is used in other places and I am not sure if it is needed for now, it will be possible to remove it when we completely switch to the new course backend